database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/source/DBSource.java

```diff
@@ -136,20 +136,24 @@ private String getJDBCPluginType() {
    */
   @Path("getSchema")
```

Comment:
this is unrelated to this PR, but ``getSchema`` seems inappropriate for a REST like endpoint. Shouldn't it just be ``schema``, and ``get`` is reflected in the Http Method?